### ISSUE TITLE
fix an issue when auto open report in WSL

### ISSUE THREAD

**Issue by ivan-toriya** [elementary-data/elementary#434]
[Slack Thread](https://elementary-community.slack.com/archives/C02CTC89LAX/p1669034786609089?thread_ts=1669034786.609089&cid=C02CTC89LAX)

"Hi, im new to using Elementary, i've set up all the steps and when i try to run: edr monitor report i get this:

The error msg is a little bit strange since it says it cannot find the file specified. But elementary has generated a elementary.html file which is shown in the screenshot."

![image](https://user-images.githubusercontent.com/43750521/204319272-803cdead-7411-42fd-b738-7dc757b2c492.png)

**Comment by ivan-toriya** [comment]
@elongl suggested
"we attempt to open it and if it fails to log it and continue so that it doesn't raise an error.
This is what [dbt does](https://github.com/dbt-labs/dbt-core/blob/43e3fc22c4eae4d3d901faba05e33c40f1f1dc5a/core/dbt/task/serve.py#L34) as well.
...
So basically no need to deal with WSL specifically.
...
The code that needs to be changed is [here](https://github.com/elementary-data/elementary/blob/ac8e4f92bda34ada1a2e168502b96acd9f8441ea/elementary/monitor/data_monitoring.py#L242).
Basically I think what we should do is wrap it in a try..except webrowser.Error and log if an occurs."

**Comment by elongl** [comment]
Thanks a lot for opening this issue.
Looking forward to the pull request.

**Comment by elongl** [comment]
Solved in https://github.com/elementary-data/elementary/pull/447.
